### PR TITLE
CERD-212 > Global Theming Approach

### PR DESCRIPTION
**Pull request by drochgenius**
**WIP**

### CODE REVIEW

File: packages/drag-drop/src/components/drag-drop.ts [hunk 1]
```diff
@@ -14,6 +14,19 @@ export class DragDrop extends ComponentBase<string[]> {
...
+        .option-item {
+            cursor: grab;
+        }
+
+        .hide {
+            visibility: hidden;
+        }
+    </style>
+    `;
+
```
Comment (drochgenius) [inline packages/drag-drop/src/components/drag-drop.ts]: Hi @adjamenhmh , I have finished cleaning up the styles for Drag & Drop.
Would you please review?
Happy to walk you through what I did if it's not clear.
But all seems to work.

File: packages/drag-drop/src/components/drop-container.ts [hunk 2]
```diff
@@ -67,6 +65,9 @@ export class DropContainer extends ComponentBase<string[]> implements Feedback {
             slot.assignedNodes().forEach(
                 (el: HTMLElement): void => {
                     if (el.classList.contains('option-item')) {
+                        if (this.sticky){
+                            el.style.position = 'absolute';
+                        }
```
Comment (drochgenius) [inline packages/drag-drop/src/components/drop-container.ts]: I found that `sticky` can only be controlled from outside with CSS. So I have done it using Javascript code.

**Comment by adjamenhmh** [comment]
@drochgenius  When the drag is initiated the option is not hidden anymore

**Comment by drochgenius** [comment]
Not sure what you mean @adjamenhmh , can you please show me?

File: packages/plot-graph/src/components/plot-graph.ts [hunk 3]
```diff
@@ -212,15 +201,26 @@ export class PlotGraph extends ComponentBase<any> {
...
+            const scale = this.scale(axis.direction, parseInt(min), parseInt(max), /* istanbul ignore next */ isDirectionX ? width : height);
 
             this.svgContainer
                 .append('g')
-                .attr('class', isDirectionX ? 'x-axis' : 'y-axis')
-                .attr('transform', isDirectionX ? translationX : translationY)
-                .call(isDirectionX ? axisBottom(scale) : axisLeft(scale));
+                .attr('class', /* istanbul ignore next */ isDirectionX ? 'x-axis' : 'y-axis')
+                .attr('transform', /* istanbul ignore next */ isDirectionX ? translationX : translationY)
+                .call(/* istanbul ignore next */ isDirectionX ? axisBottom(scale) : axisLeft(scale));
```
Comment (drochgenius) [inline packages/plot-graph/src/components/plot-graph.ts]: Hey @chrisunl , this is how I ignored the ternary if-else from coverage reporting.

I have updated all tests to 100% coverage.

**Comment by adjamenhmh** [comment]
@drochgenius  feedback is not working on my end

File: packages/drag-drop/src/components/drag-drop.ts [hunk 4]
```diff
@@ -58,7 +68,8 @@ export class DragDrop extends ComponentBase<string[]> {
             }
             if (!(this.currentElement.parentElement as DragContainer).dispenser) {
                 setTimeout(function() {
```
Comment (drochgenius) [inline packages/drag-drop/src/components/drag-drop.ts]: Hi @adjamenhmh , this leads to a really weird behaviour when I execute the code within a Problem.
The whole problem gets hidden while dragging!

I'd like to clarify with you what was the problem you were trying to solve with the setTimeout

**Pull request merged**